dex/src/main/scala/com/wavesplatform/dex/cli/WavesDexCli.scala

```diff
@@ -284,6 +286,151 @@ object WavesDexCli extends ScoptImplicits {
     removed.get()
   }
 
+  def inspectAsset(args: Args): Unit =
+    for {
+      _ <- cli.log(
+        s"""
+           |Passed arguments:
+           |  DEX config path : ${args.configPath}
+           |  Asset id        : ${args.assetId}
+           |""".stripMargin
+      )
+      settings = loadMatcherSettingsFromPath(args.configPath)
+      assetIdBytes <- ByteStr.decodeBase58(args.assetId).toEither
+    } yield withLevelDb(settings.dataDirectory) { db =>
+      AssetsDb.levelDb(db).get(IssuedAsset(assetIdBytes)) match {
+        case None => println("There is no such asset")
+        case Some(x) =>
+          println(
+            s"""
+               |Decimals   : ${x.decimals}
+               |Name       : ${x.name}
+               |Has script : ${x.hasScript}
+               |Is NFT     : ${x.isNft}
+               |""".stripMargin
+          )
+      }
+    }
+
+  def mockAsset(args: Args): Unit = {
+    val name = if (args.name.isEmpty) args.assetId else args.name
+    for {
+      _ <- cli.log(
+        s"""
+           |Passed arguments:
+           |  DEX config path : ${args.configPath}
+           |  Asset id        : ${args.assetId}
+           |  Name:           : ${args.name}
+           |     Will be used : $name
+           |  Decimals        : ${args.decimals}
+           |  Has script      : ${args.hasScript}
+           |  Is NFT          : ${args.isNft}
+           |""".stripMargin
+      )
+      settings = loadMatcherSettingsFromPath(args.configPath)
+      assetIdBytes <- ByteStr.decodeBase58(args.assetId).toEither
+    } yield withLevelDb(settings.dataDirectory) { db =>
+      val briefAssetDescription = BriefAssetDescription(
+        name = name,
+        decimals = args.decimals,
+        hasScript = args.hasScript,
+        isNft = args.isNft
+      )
+
+      println(s"Writing $briefAssetDescription...")
+      AssetsDb.levelDb(db).put(IssuedAsset(assetIdBytes), briefAssetDescription)
+    }
+  }
+
+  def listAssetPairs(args: Args): Unit =
+    for {
+      _ <- cli.log(
+        s"""
+           |Passed arguments:
+           |  DEX config path : ${args.configPath}
+           |""".stripMargin
+      )
+      settings = loadMatcherSettingsFromPath(args.configPath)
+    } yield withLevelDb(settings.dataDirectory) { db =>
+      val assetPairs = AssetPairsDb.levelDb(db).all().toVector.sortBy(_.key)
+      if (assetPairs.isEmpty) println("There are no order books")
+      else {
```

Comment:
`listAssetPairs` prints that there are no orderbooks)

Reply:
Fixed